static/src/javascripts/test/spec/common/commercial/gustyle/gustyle.spec.js

```diff
@@ -0,0 +1,65 @@
+import Injector from 'helpers/injector';
+import $ from 'common/utils/$';
+import fastdom from 'fastdom';
+import fixtures from 'helpers/fixtures';
+
+var Sut, // System under test
+    injector = new Injector(),
+    gustyle;
```

Comment:
Now that Node 4.0 supports ES6, maybe we should start using `let` and `const` rather than var.